#ifndef HARDWARE_CONTROL_INTERFACE_H_
#define HARDWARE_CONTROL_INTERFACE_H_

#include <ros/ros.h>
#include <serial/serial.h>
#include <hci/sensorValue.h>
#include <hci/motorCommand.h>
#include <hci/driveCommand.h>
#include <map>
#include <string>

using std::map;
using std::vector;
using std::string;



#endif